src/randomStringGenerator.md

```diff
@@ -73,3 +74,28 @@ generateString(){
     echo $result
 }
 ```
+
+### Cpp
+```cpp
+using namespace std;
```

Comment:
see if u can create the function without external dependency _( If this is coming as external )_